packaging/rpm/ansible-tower-cli.spec

```diff
@@ -0,0 +1,53 @@
+%{!?python_sitelib: %define python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print get_python_lib()")}
+
+Name:           ansible-tower-cli
+Version:        2.3.1
```

Comment:
note to self for later - we'll add this to the list of things to update pre-release